godtossing/Pos/Ssc/GodTossing/Core/Types.hs

```diff
@@ -22,7 +22,7 @@ module Pos.Ssc.GodTossing.Core.Types
        , SharesDistribution
 
          -- * Vss certificates
-       , VssCertificate (vcVssKey, vcExpiryEpoch, vcSignature, vcSigningKey)
+       , VssCertificate (..)
```

Comment:
It's totally bad. Constructor of `VssCertificate` shouldn't be exported, it's unsafe.